questions.md

```diff
@@ -0,0 +1,3 @@
+Could you tell difference between pull and fork?
+
+Can you explain about fork?
```

Comment:
Sure! Forks are copies of a repository. Here is more info: https://help.github.com/articles/fork-a-repo/